homeassistant/core.py

```diff
@@ -749,16 +749,17 @@ class Service(object):
     """Representation of a callable service."""
 
     __slots__ = ['func', 'description', 'fields', 'schema',
-                 'is_callback', 'is_coroutinefunction']
+                 'is_callback', 'is_coroutinefunction', 'state_to_set']
```

Comment:
Also, should this be the value of the state, eg `'on'` or a `State` instance? RIght now I settled for the value, and did not include state attributes here. The schema will give information what attributes are required anyway.